### DISCUSSION TITLE
Account suspended, support not responding

### DISCUSSION THREAD

**Discussion by ghost** [community/community#128933] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My github accoount had also been suspended for some violation (@D4ZA1) and i have not got any mail regarding what the violation was. I tried reaching out to support. but all i got was "Thank you for contacting GitHub Support. We wanted to let you know that we've received your message. We are experiencing high volumes and therefore, you may experience longer than normal wait times. In the meantime, you may find answers to commonly asked questions in our [community forum](https://github.community/) or in our [documentation](https://docs.github.com/).". and it has been 3 weeks, i tried replying to the message, raising new tickets but it doesnt allow multiple tickects. It says that my ticket was updated 3 weeks ago but i dont see any such thing. I have no idea when they will reply

**Comment by LiteBrite82** [comment]
Hi @H30BR1N3 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Reply by ghost** [reply]
ok, thankyou
